app/src/main/java/org/fossasia/susi/ai/chat/ParseSusiResponseHelper.kt

```diff
@@ -78,6 +81,34 @@ class ParseSusiResponseHelper {
                 Timber.e(e)
             }
 
+            Constant.TABLE -> try {
+                count = 0
+                val listColumn: ArrayList<String> = ArrayList()
+                val listColVal: ArrayList<String> = ArrayList()
+                val listTableData: ArrayList<String> = ArrayList()
```

Comment:
val anything = ArrayList<String>()

Reply:
@iamareebjamal please suggest what should be done here